src/commontools.c

```diff
@@ -100,17 +108,17 @@ btc_bool hd_print_node(const btc_chainparams* chain, const char* nodeser)
         return false;
 
     size_t strsize = 128;
-    char str[strsize];
+    char str[128];
```

Comment:
Note: duplicate specification of constant 128 here.  My style if I merge might be to change uses of `strsize` to `sizeof(str)`.

Reply:
This is not a constant duplication - this is C99 conformance. The arrays in C can't have dimension specified by a variable (and VS produce a compiler error) - so it should be either a #define, or just a numeric.